src/app/work-item/work-item.component.html

```diff
@@ -29,8 +29,29 @@ <h2 class="card-pf-title text-center">
     <div class="row toolbar-pf">
       <div class="col-md-12 col-sm-12 col-xs-12">
         <div class="list-action toolbar-pf-actions">
+          <!--Filter-->
+          <div class="form-group toolbar-pf-filter">
+            <label class="sr-only" for="filterwitype">Filter</label>
+            <div class="input-group">
+              <div class="input-group-btn" dropdown>
+                <button class="btn btn-default dropdown-toggle" type="button" id="wi_filter_dropdown" data-toggle="dropdown" dropdown-open>
+                  Filter
+                  <span class="caret"></span>
+                </button>
+                <ul class="dropdown-menu" role="menu" aria-labelledby="wi_filter_dropdown">
+                  <li *ngIf="!filters.length">
+                    <span>No filters available</span>
+                  </li>
+                  <li *ngFor="let filter of filters;" role="presentation" (click)="activeFilter(filter.id)">
```

Comment:
The width of "Assign to me" needs to match the wireframe:
![filterscreenshot1](https://cloud.githubusercontent.com/assets/1892722/22678822/0184d808-ed23-11e6-8f88-3140edd39f99.png)
On UI:
![filterdropdownui](https://cloud.githubusercontent.com/assets/1892722/22678847/24b039da-ed23-11e6-8682-d9ae972ba7ec.png)